### DISCUSSION TITLE
Genesis mismatch

### DISCUSSION THREAD

**Discussion by jalw0tpyrc** [gochain-io/docs#145] [Q&A]
Hi, 
When running the testnet and mainnet we are getting genesis mismatch, following the docs genesis and used the genesis-main.json and genesis-testnet, no success. 

```
DEBUG[09-05|11:09:26] GoChain handshake failed                 id=2f5d0489e2bbbc49 conn=dyndial         err="Genesis block mismatch - b8968a99546e385a (!= 5b44a92a842a888b)"
TRACE[09-05|11:09:26] Protocol eth/63 failed                   id=2f5d0489e2bbbc49 conn=dyndial         err="Genesis block mismatch - b8968a99546e385a (!= 5b44a92a842a888b)"
```

Could we get the genesis for both testnet and mainnet?

**Comment by treeder** [comment]
Dupe: https://github.com/gochain/gochain/discussions/459
